sidekick_core/test/commands/plugins/create_plugin_command_test.dart

```diff
@@ -13,34 +11,4 @@ void main() {
 
     expect(CreatePluginCommand.templates.keys, expectedTemplates);
   });
-
-  group('generates valid plugin code', () {
```

Comment:
This test file is now quite empty, can we move `test('check available templates'`?